common/lib/xmodule/xmodule/templates/html/raw.yaml

```diff
@@ -3,12 +3,10 @@ metadata:
     display_name: Raw HTML
     editor: raw
 data: |
-      <p>This template is similar to the Text template. The only difference is 
-      that this template opens in the Raw HTML editor rather than in the Visual
-      editor.</p>
+      <p>This template is similar to the Text template. The only difference is
```

Comment:
Note: We should Not Merge this until we turn on the text editor, as it changes the info.